tools/awssdkpatch/patch.tmpl

```diff
@@ -59,18 +59,22 @@ var x identifier
 
 # Step 2: Convert AWS SDK v1 (service) to v2
 @@
-var x identifier
+var i, x identifier
 @@
--import "github.com/aws/aws-sdk-go/service/{{ .GoV1Package }}"
-+import "github.com/aws/aws-sdk-go-v2/service/{{ .GoV2Package }}"
-{{ .GoV1Package }}.x
+-import i "github.com/aws/aws-sdk-go/service/{{ .GoV1Package }}"
++import i "github.com/aws/aws-sdk-go-v2/service/{{ .GoV2Package }}"
+{{ .ImportAlias }}.x
 
 # 2a: Most types are in the service's types package so do a simple mass-substitution
 # to get most things right
 @@
 var x identifier
 @@
+{{- if eq .ImportAlias "" }}
 -{{ .GoV1Package }}.x
+{{- else }}
+-{{ .ImportAlias }}.x
+{{- end }}
 +awstypes.x
```

Comment:
Is this condition possible, given `ImportAlias` is set to `GoV2Package` when empty during `getProviderData`?

```suggestion
-{{ .ImportAlias }}.x
+awstypes.x
```

Reply:
Good catch; that was a remnant of an earlier approach I'd taken.